onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -40,14 +40,15 @@ def __init__(self, model_path, data_reader: CalibrationDataReader, calibrate_op_
         :param op_types: operator types to be calibrated and quantized, default = 'Conv,MatMul'
         :param black_nodes: operator names that should not be quantized, default = ''
         :param white_nodes: operator names that force to be quantized, default = ''
+        :param augment_all: augment all FP32 activation tensors, default = False
```

Comment:
@yufenglee , what's your take on this api? i think we are trying to provide flexibility to user to specify which nodes to quantize.  
so far, the api supports by op type, by node name,  now we want to give option by data type /all option.
op_types, black_nodes, white_nodes arguments all work and are consistent with each other.
But i'm not sure about augment_all.
meaning, in some sense if you see augment_all,  op_types, black_nodes etc. all carry no meaning and should be ignored?
is there a better way to make it all consistent with each other?

Reply:
what if the default behavior could mean augment all?
i.e. empty op_types, empty black_nodes, and empty white_nodes

Reply:
Everything empty means no augmentation. We could add all nodes to the white_nodes list for TRT, but it requires user to get graph's node list first before calibration.

Reply:
it doesn't need to mean that. the default could mean to augment everything, unless you explicitly specify which ops or node names to include/exclude.
i don't think it really makes sense for empty to mean no augmentation. why would they call the api if there is nothing to do.